dist/less/overlay-panel.less

```diff
@@ -89,6 +103,9 @@ body.overlay-open {
     height: @wizard-pf-steps-indicator-height;
     margin: @wizard-pf-steps-indicator-margin-top-bottom 0;
     padding: 0;
+    @media(max-width: @screen-sm-min) {
+      margin-top: calc(@wizard-pf-steps-indicator-margin-top-bottom + 10px);
```

Comment:
But if there are more steps then eventually it runs over. Will happen in non-mobile as well if there are lots of steps so I was leaning more towards always having the extra margin so that it would never overlap.